source/css/sass/patterns/molecules/teaser.scss

```diff
@@ -0,0 +1,30 @@
+@import "../../mixins/spacing";
+@import "../../mixins/typography";
+
+.teaser-list {
+
+  .teaser__header {
+    @include block-spacing($start: 0);
+  }
+
+  .teaser-list__list {
+    @include padding(0, inline-start);
+  }
+
+  .teaser-list__item {
+    @include list-style-none();
+
+    &:not(:first-child) {
+      .teaser {
+        @include padding($baselinegrid-space-small, block-start);
+        @include divider(block-start);
```

Comment:
We should use end margins (see #66). And these probably apply to the list item rather than the teaser itself (otherwise should be styled in that pattern).

Reply:
👍 to end margins.

There was a reason we applied the space to the teaser and not the list when building eLife 2. Rather vexingly, I can't remember what that reason was though.